cfgov/v1/jinja2/v1/story_page.html

```diff
@@ -26,6 +26,10 @@
 {%- endblock %}
 
 {% block content_main %}
+
+{% import 'v1/includes/molecules/translation-links.html' as translation_links with context %}
+{{ translation_links.render(modifier_classes='block--flush-top') }}
```

Comment:
Prolly indent this to align with the for loop below, although we're not consistent with whether to indent or not in blocks.

Reply:
done in https://github.com/cfpb/consumerfinance.gov/pull/8397/commits/548663489710ca1df5e643950ce3a2dd3c37f7ed!